### PR TITLE
#5 Feature/5 Reserva Plaza

### PR DESCRIPTION
**Pull request by ferandgal**
Cambiada los endpoints por los de heroku

### CODE REVIEW

File: src/Plaza/getPlazas.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export default async function getPlaza() {
+    //const apiURL = 'https://park-inn-ispp-be.herokuapp.com/plazas/all';
+    return await fetch('http://localhost:8080/plazas/all', {
+        method: 'GET',
+        credentials:'same-origin',
```
Comment (luibelzan) [inline src/Plaza/getPlazas.js:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
        credentials:"same-origin",
```

File: src/Plaza/getPlazas.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export default async function getPlaza() {
+    //const apiURL = 'https://park-inn-ispp-be.herokuapp.com/plazas/all';
+    return await fetch('http://localhost:8080/plazas/all', {
+        method: 'GET',
```
Comment (luibelzan) [inline src/Plaza/getPlazas.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
        method: "GET",
```

File: src/Plaza/getPlazas.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        credentials:'same-origin',
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+        }
+    })
+        .then(res => res.json())
+        .then(plaza => console.log(plaza))
```
Comment (luibelzan) [inline src/Plaza/getPlazas.js:12]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
        .then((plaza) => console.log(plaza))
```

File: src/Plaza/getPlazas.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export default async function getPlaza() {
+    //const apiURL = 'https://park-inn-ispp-be.herokuapp.com/plazas/all';
+    return await fetch('http://localhost:8080/plazas/all', {
+        method: 'GET',
+        credentials:'same-origin',
+        headers: {
+            'Accept': 'application/json',
```
Comment (luibelzan) [inline src/Plaza/getPlazas.js:7]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
            "Accept": 'application/json',
```

File: src/Plaza/getPlazas.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+            'Content-Type': 'application/json',
+        }
+    })
+        .then(res => res.json())
+        .then(plaza => console.log(plaza))
+        .catch(error => console.error("Error: ",error));
```
Comment (luibelzan) [inline src/Plaza/getPlazas.js:13]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
        .catch((error) => console.error("Error: ",error));
```

File: src/Plaza/getPlazas.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+export default async function getPlaza() {
+    //const apiURL = 'https://park-inn-ispp-be.herokuapp.com/plazas/all';
+    return await fetch('http://localhost:8080/plazas/all', {
```
Comment (luibelzan) [inline src/Plaza/getPlazas.js:3]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
    return await fetch("http://localhost:8080/plazas/all", {
```

File: src/components/Calendario.elements.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
+import { colors } from '../theme';
```
Comment (luibelzan) [inline src/components/Calendario.elements.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
import { colors } from "../theme";
```

File: src/Plaza/ReservaPlaza.elements.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
```
Comment (luibelzan) [inline src/Plaza/ReservaPlaza.elements.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
import styled from "styled-components"
```

File: src/Plaza/ReservaPlaza.elements.js [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+import { colors } from '../theme';
```
Comment (luibelzan) [inline src/Plaza/ReservaPlaza.elements.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
import { colors } from "../theme";
```

File: src/components/Calendario.elements.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
```
Comment (luibelzan) [inline src/components/Calendario.elements.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
import styled from "styled-components"
```

**Review by mangarmar — APPROVED** [review]

**Pull request merged**